### PR TITLE
account operations

### PR DESCRIPTION
**Pull request by evakule**

### CODE REVIEW

File: src/test/java/com/somn/SomnApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.somn;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class SomnApplicationTests {
+
+//  @Test
+//  void contextLoads() {
```
Comment (ssuchkov) [inline src/test/java/com/somn/SomnApplicationTests.java]: Please add unit test or integration test for CRUD

File: src/main/resources/sql/init.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+CREATE TABLE IF NOT EXISTS FK_USER_ROLE (
+    id              BIGINT AUTO_INCREMENT PRIMARY KEY NOT NULL,
+    user_id         BIGINT NOT NULL,
+    role_id         BIGINT NOT NULL,
+    CONSTRAINT      FK_USER_ID_USERS FOREIGN KEY (user_id)
+      REFERENCES    users(id),
+    CONSTRAINT      FK_ROLE_ID_ROLES FOREIGN KEY (role_id)
+      REFERENCES    roles(id)
+);
+
```
Comment (ssuchkov) [inline src/main/resources/sql/init.sql]: Please split setting up model and test values into two files

File: README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Simple online banking app
```
Comment (ssuchkov) [inline README.md]: Please add local setup information into the readme
-required technologies (JVM, DB, etc.)
-start/stop commands

File: src/main/java/com/somn/model/AccountEntity.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ * @version 1.0
+ */
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Table(name = "accounts")
+public class AccountEntity extends BaseEntity {
```
Comment (ssuchkov) [inline src/main/java/com/somn/model/AccountEntity.java]: Please split into DTO and Entity layers having proper names
AccountDTO and AccountEntity

File: src/main/resources/sql/init.sql (hunk 2 above)
Comment (evakule) [reply src/main/resources/sql/init.sql]: done

File: src/main/java/com/somn/model/AccountEntity.java (hunk 4 above)
Comment (evakule) [reply src/main/java/com/somn/model/AccountEntity.java]: not necessary for now

File: src/test/java/com/somn/SomnApplicationTests.java (hunk 1 above)
Comment (evakule) [reply src/test/java/com/somn/SomnApplicationTests.java]: done

File: README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+Set up your database properties as below
+    
+    datasource.url = localhost:3306/sonm
+    datasource.username = root
+    datasource.password = 12345
+    
+- Secondly, create database 'sonm' and run next commands in it:
+
+
```
Comment (ssuchkov) [inline README.md]: you can refer to your existing scripts instead of copy paste the queries

**Pull request merged**